src/main/java/com/uber/cadence/internal/worker/PollerOptions.java

```diff
@@ -133,6 +138,11 @@ public Builder setPollThreadNamePrefix(String pollThreadNamePrefix) {
       return this;
     }
 
+    public Builder setPollOnlyIfHasCapacity(boolean pollOnlyIfHasCapacity) {
```

Comment:
let's add a comment here to describe what it does, e.g.
```
the poller will check task executor's remaining capacity before polling tasks. This is to prevent task to get started but not being able to execute in time. 
```

Reply:
added comment